### ISSUE TITLE
New preset: shop=ski

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7880]
A shop selling skis and related accessories and services.
https://wiki.openstreetmap.org/wiki/Tag:shop%3Dski

**Comment by quincylvania** [comment]
Related: #7625

**Comment by simonpoole** [comment]
The popular tagging scheme for shops selling/renting/servicing sport goods is shop=sports plus a (potentially multi-valued) sport=* tag. Suddenly splitting these things all out in to separate shop values doesn't make sense (not to forget typically that the shops do actually cater for more than one activity to start with).

**Comment by maro-21** [comment]
It's not mentioned on Wiki.
For me `shop=sports` isn't the same as `shop=golf`, `shop=boat` or `shop=ski`.
Shop=sports sells mainly clothes and some accessories for popular everyday sports like: running, soccer, basketball and these shops are popular in shopping malls in my country. I won't buy a golf club there or a boat.
Shops shop=boat are common usually in regions close to water (sea, lake, ocean) and selling sailing equipment. And I won't buy a ball for basketball there.

For the same reason you could say that we don't need shop=appliance, shop=camera, shop=computer, shop=hifi, shop=mobile_phone becasue we have shop=electronics.
Or you could say that we don't need shop=cheese, shop=frozen_food, shop=wine, shop=spices because there is shop=supermarket and we can use an additional tags for things they sell. They're not the same. Shop=sports it's like shop=supermarket for sports shops.

**Comment by simonpoole** [comment]
The difference is that shop=sports + sport=* is reasonably popular, whereas the "specialized" values have minuscule use except in the cases where there is actually a clearly well defined shop type.

Further using the example at hand, shop=ski is at odds with reality, as these shops (if not changing seasonally to start with) will typically offer a range of winter sports related goods from alpine skiing over cross country, snow shoes to skating. If open outside of Winter they will typically switch to outdoor gear.

So if anything more specific  than shop=sports it should be winter_sports or similar.

**Comment by Adamant36** [comment]
> The difference is that shop=sports + sport=* is reasonably popular

So are a lot of general tags until more specific ones come along. 

> Further using the example at hand, shop=ski is at odds with reality, as these shops (if not changing seasonally to start with) will typically offer a range of winter sports related goods from alpine skiing over cross country, snow shoes to skating. If open outside of Winter they will typically switch to outdoor gear.

It would be extremely wrong IMO to claim there aren't some shops that just sell skis. It's completely nonsensical to go with a plural tag "shop=sports" for places that sell items of a single sport. 

Also, the Wiki is pretty explicit that when there are usable top level tags for something, that it is preferably to go with the top level tag (in this case shop=*) by splitting the feature into multiple POIs. Instead of using a semi-colon value separator to show that the shop sells multiple things. Sports=Ski isn't a top level tag though and it forces people to use a  semi-colon value separator when it might be the best thing way to tag things. Since there's plenty of instances where it is better to split the shop into separate POIs as the Wiki suggests. For instances when the ski shop has separate opening hours, contact information, etc etc from the main store. Otherwise, by combining them your potentially losing important information (just like with not having a amenity=ski_rental preset BTW).

 Even in cases where you aren't losing information though, it's still perfectly legitimate to map a store selling multiple things as separate POIs and like I said it's exactly what the Wiki says to do. Last I checked, we should all be for following the Wiki.

**Comment by simonpoole** [comment]
No, the wiki is descriptive, not prescriptive.

**Comment by Adamant36** [comment]
> No, the wiki is descriptive, not prescriptive.

Obviously. In this specific case though I assume its describing something that is established consensus and clearly a "standard" by virtue of being how most things are usually mapped etc.  That's not to say everything contained in the wiki is defacto information that should be followed without question, but I think defering to it in this partuclar case is perfectly fine. 

Again, 99% of the time POIs are mapped separately when they can be. For instance gas stations and convience stores, pharmacies and chemists etc etc. So its not some revolutionary idea, that isn't based in how people already map or anything.
